plugins/CoreAdminHome/tests/UI/WhatIsNew_spec.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Screenshot integration tests.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("WhatIsNew", function () {
+    this.timeout(0);
+    this.fixture = 'Piwik\\Tests\\Fixtures\\CreateChanges';
+
+    before(function () {
+      testEnvironment.optionsOverride = {
+          loadChanges: '1'
+      };
+
+      testEnvironment.overrideConfig('General', {
+        enable_internet_features: 0
+      });
+
+      testEnvironment.save();
+    });
+
+    it('should show the what is new changes popup', async function() {
+        await page.goto('?module=CoreHome');
+        await page.$('.whatisnew');
+        await page.waitForTimeout(1000);
```

Comment:
Instead of waiting for a fixed number seconds it would make more sense to wait for certain elements to become visible.
```suggestion
        await page.waitForSelector$('.whatisnew', {visible: true});
```

Reply:
This test gave me some grief to get working reliably so I'd carefully backed away from changing it once it was reliable! :smile:  
This change seems to also work well though and anything that removes fixed timeouts has to be good!

Reply:
The changes caused the screenshot to be taken before the background widgets had loaded, so it looks like it could be unreliable. I've added a network idle timeout to see if that helps.

Reply:
To avoid that, you could also only take a screenshot of the overlay only. And if the icon in the nav bar should be checked as well you could add a second test to take a screenshot of that part only. That would make the test fully independent from anything loaded in the background.